qutebrowser/mainwindow/prompt.py

```diff
@@ -331,9 +331,14 @@ def _on_show_prompts(self, question):
 
         if not question.interrupted:
             # If this question was interrupted, we already connected the signal
-            question.aborted.connect(
-                lambda: modeman.leave(self._win_id, prompt.KEY_MODE, 'aborted',
-                                      maybe=True))
+            def on_aborted():
```

Comment:
(more of a question to @The-Compiler, but) why are we leaving prompt mode when question is aborted here? There is no dedicated command for aborting prompts, instead it's tied to `:leave-mode`. For windows that aren't the one currently being aborted, they should be handled by the `_on_global_mode_left` method below. 

It's only purpose seems (to me, at least) to be changing the message in the log to 'aborted' (for tests) which dosen't seem necessary to me.

I would vote for just removing this connection completely and adjusting the tests, this solution feels too hacky to me.